tests/test_main.py

```diff
@@ -1,14 +1,20 @@
-"""Tests main module of kytos/kronos."""
-from flask import Flask
-from tests.helpers import get_controller_mock
-
-from napps.kytos.kronos.utils import NamespaceError
+"""Module to test main module of kytos/kronos.
 
-# pylint: disable=wrong-import-order,wrong-import-position
+Unit tests to check fail and success cases in main module.
+isort:skip_file
```

Comment:
Can this be a comment instead of a docstring line?

Reply:
No, I've tested and doesn't work and this how the documentation specify it.